npm-shrinkwrap.json

```diff
@@ -1,6159 +1,8 @@
 {
   "name": "@pnp/cli-microsoft365",
   "version": "3.11.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
It seems like you're using npm@6, whereas our shrinkwrap file is generated with npm@7. Could you upgrade to npm@7 and reinstall the Graph types package and then recommit the shrinkwrap? Thank you!

Reply:
@waldekmastykarz sure, it also seems that there are issues with the coverage for "ubuntu-latest, 14, 14". I will look into it soon.

Reply:
Done. Also, added tests for the coverage issue.